cdhweb/people/views.py

```diff
@@ -195,17 +195,16 @@ def get_past_profiles(self):
             .order_by('-most_recent')
 
 
-class FacultyListView(ProfileListView):
-    '''Display current and past faculty affiliates'''
-    page_title = 'Faculty Affiliates'
+class AffiliateListView(ProfileListView):
```

Comment:
Does anything need to happen to migrate/convert the existing mezzanine page/placeholder in menus for the current faculty profile page when this change takes effect?

Reply:
just to rename it "affiliates" and edit its text content slightly; I didn't know if that was worth a migration - seems like not.

Reply:
should this be a `DEPLOYNOTES` thing, or is it fine to not address?

Reply:
I think useful to document in deploy notes just to make sure we don't forget